src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -144,6 +146,10 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       template = <div>Unknown Template: ${(venue as AnyVenue).template}</div>;
   }
 
+  if (HAS_RANDOMIZED_POSTERS_TEMPLATES.includes(venue.template)) {
+    template = <PostersProvider venueId={venue.id}>{template}</PostersProvider>;
+  }
+
```

Comment:
We shouldn't be mutating/reassigning the `template` var here like this.

Now that we are using `HAS_RANDOMIZED_POSTERS_TEMPLATES`, just put the rest of this back in the JSX where it was before. There was no need to extract it out here like this.

Reply:
Actually.. thinking about this more.. `HAS_RANDOMIZED_POSTERS_TEMPLATES` is definitely not the right name for this.. If you try and render any of the venues that relies on `PostersProvider` when it's not present it will throw an error. So this has nothing to do with whether they should be randomised, and everything to do with poster provider itself.

Reply:
Just call it something like `POSTER_PROVIDER_TEMPLATES` or similar.

Reply:
renamed b4c9d461ba2e3bb154e34624998df49e1c40bf73